Content.Shared/Projectiles/ProjectileSpreadComponent.cs

```diff
@@ -0,0 +1,32 @@
+using Content.Shared.Weapons.Ranged.Systems;
+using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared.Projectiles;
+
+/// <summary>
+/// Spawns a spread of the projectiles when fired
+/// </summary>
+[RegisterComponent, NetworkedComponent, Access(typeof(SharedGunSystem))]
+public sealed partial class ProjectileSpreadComponent : Component
```

Comment:
This component is unused. I don't see why it's here.

Reply:
it honestly shouldn't be, I could've sworn I removed it